docs/development_tips.md

```diff
@@ -16,7 +16,7 @@ See the [API library docs](api_lib_index#trying-your-library-inside-home-assista
 
 ### Test Core integration changes in your production Home Assistant environment
 
-If you made a change to a core integration and you want to test it in your production Home Assistant environment, copy the integration folder into `/config/custom_components`, add a version to the `manifest.json` (e.g. "version": "0.0"), and restart Home Assistant. Home Assistant will always prioritize integrations in `custom_components` over the core integration. Don't forget to remove it once you are done testing; otherwise, you will be stuck on that version.
+If you made a change to a core integration and you want to test it in your production Home Assistant environment, copy the integration folder into `/config/custom_components`, add a version to the `manifest.json` (e.g. "version": "0.0.0"), and restart Home Assistant. Home Assistant will always prioritize integrations in `custom_components` over the core integration. Don't forget to remove it once you are done testing; otherwise, you will be stuck on that version.
```

Comment:
If this is true, that is a bug.
Do you have logs that state that "x.x" is not valid?

Reply:
`"version": "0.0"` works just fine for me (even ` "version": 0.0` (no quote) works) on current dev.
Are you sure you saved the file?

Im fine with the change here, I just want to figure out if the reason was a bug or something else.

Reply:
It very likely could have been user error on my part. If you can reproduce it working properly then I wouldn't worry about it.